### PR TITLE
Print also the tracking branch in git status

### PR DESCRIPTION
**Pull request by S-Dafarra**

### CODE REVIEW

**Review by traversaro — APPROVED** [review]

**Comment by traversaro** [comment]
Thanks!

File: scripts/robotologyGitStatus.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ printGitStatus () {
     (cd $1 && echo -n "${1}: " \
     && git rev-parse --abbrev-ref HEAD \
     && git log -1 --format="%cr|%s" \
-    && git status --short); \
+    && git status -sb --short); \
```
Comment (drdanz) [inline scripts/robotologyGitStatus.sh:30]: It's already merged, but `-s` and `--short` are actually the same flag...

Comment (S-Dafarra) [reply scripts/robotologyGitStatus.sh:30]: But ``--short`` and ``-sb`` are not the same :grin:

Comment (traversaro) [reply scripts/robotologyGitStatus.sh:30]: Feel free to fix!

Comment (S-Dafarra) [reply scripts/robotologyGitStatus.sh:30]: Here https://github.com/robotology/robotology-superbuild/pull/559

**Pull request merged**